### PR TITLE
Add uncountable word from Laravel Framework

### PR DESCRIPTION
**Pull request by cesarep**
Simply adds uncountable words from [Laravel](https://github.com/laravel/framework/blob/9.x/src/Illuminate/Support/Pluralizer.php) not presented here: `'cattle', 'kin', 'recommended', 'related'`

### CODE REVIEW

File: lib/Doctrine/Inflector/Rules/English/Uninflected.php [hunk 1]
```diff
@@ -149,6 +151,8 @@ private static function getDefault(): iterable
         yield new Pattern('progress');
         yield new Pattern('rabies');
         yield new Pattern('rain');
+        yield new Pattern('recommended');
+        yield new Pattern('related');
```
Comment (malarzm) [inline lib/Doctrine/Inflector/Rules/English/Uninflected.php]: these two aren't nouns, why are they being pluralized somewhere?

Comment (cesarep) [reply lib/Doctrine/Inflector/Rules/English/Uninflected.php]: Laravel uses it, among other things, to determine the parameters name on a route, singularizing it, like: /images/{image}/, and table names for models, pluralizing it.
My guess is those are fairly common words to use in a URL, but to keep things consistent, maybe those should be handed there, and not here

Comment (malarzm) [reply lib/Doctrine/Inflector/Rules/English/Uninflected.php]: That be the way to go in my opinion. Also please add tests for other 2 words that are to stay :)

**Comment by malarzm** [comment]
Thanks @cesarep!

**Pull request merged**